eventuate-common-jdbc/src/main/java/io/eventuate/common/jdbc/EventuateCommonJdbcOperations.java

```diff
@@ -33,23 +33,22 @@ public void insertIntoEventsTable(String eventId,
   }
 
 
-  public void insertIntoMessageTable(String messageId,
-                                      String payload,
+  public long insertIntoMessageTable(String payload,
```

Comment:
With regards to supporting old and new simultaneously.

Perhaps this should return the messageId - not just the databaseId.
That would be more consistent with the original method - which took a messageId.
It would hide the details of ID generation here.

It could also take a messageId parameter (which might be null).